app/locales/en_US.json

```diff
@@ -0,0 +1,126 @@
+{
+    "menu.Undo": "Undo",
```

Comment:
I'm wondering why you don't use a more classical nested object here?

`{ "menu": { "redo": "Redo", ... } }`